KoreBuild-dotnet/build/_xunit-test.shade

```diff
@@ -34,37 +34,47 @@ projectFile=''
             var testArgs = noParallelTestProjects.Contains(projectName) ? " -parallel none" : "";
 
             var configs = project.ValueAsJsonObject("frameworks");
-            IEnumerable<string> targetFrameworks = configs == null?
-                new string[0]:
-                configs.Keys.Where(k => k.StartsWith("dnx", StringComparison.OrdinalIgnoreCase));
+            var targetFrameworks = configs == null
+                ? new string[0]
+                : configs.Keys
+                    .Where(k => k.StartsWith("dnx4", StringComparison.OrdinalIgnoreCase)
+                        || k.StartsWith("net4", StringComparison.OrdinalIgnoreCase));
 
             var runnerFolder = Path.GetFullPath(Path.Combine(KoreBuildFolderPath, "build", "xunit.runner.console", "tools"));
             var xunitCoreFolder = Path.GetFullPath(Path.Combine(KoreBuildFolderPath, "build", "xunit.core", "build", "_desktop"));
             
             foreach (var framework in targetFrameworks)
             {
-                if (!framework.StartsWith("dnxcore"))
+                if (IsLinux)
+                {
+                    // Work around issue with testing in parallel on Mono.
+                    testArgs = " -parallel none";
```

Comment:
`dotnet test` is used for tests targeting `dnxcore50` and is used in `_dotnet-test.shade`. Tests targeting dnx451 or net451 will be run using the xunit runner which requires the switch.